lib/code_corps_web.ex

```diff
@@ -41,6 +40,9 @@ defmodule CodeCorpsWeb do
 
       import Canary.Plugs
       import CodeCorpsWeb.AuthenticationHelpers, only: [load_and_authorize_changeset: 2]
+
+      alias CodeCorps.{Repo, Policy}
+      alias Plug.Conn
```

Comment:
These will presumably be used in ally controllers, once we make the full switch so no reason not to include them here.